pkg/client/dynamic/clientset/tekton/tekton.go

```diff
@@ -25,5 +32,15 @@ func WithClient(client dynamic.Interface) clientset.Option {
 				cs.Add(r, client)
 			}
 		}
+		for version, resources := range allowedTriggersTypes {
+			for _, resource := range resources {
+				r := schema.GroupVersionResource{
+					Group:    triggers.GroupName,
```

Comment:
ah I see, you had to split these apart because of the group. 👍

Reply:
Yeah....